keyboards/primekb/meridian/meridian.c

```diff
@@ -35,19 +35,19 @@ void keyboard_post_init_user(void) {
 bool led_update_kb(led_t led_state) {
```

Comment:
```suggestion
__attribute__((weak))
bool led_update_kb(led_t led_state) {
```

Reply:
@newtonapple This change isn't actually needed.  First, it's the keyboard level function, so ... it shouldn't be replaced.

Second, the `res` bool allows users to replace/disable this functionality, by returning false from `led_update_user`